tests/layer_tests/pytorch_tests/test_group_norm.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class TestGroupNorm(PytorchLayerTest):
+    def _prepare_input(self):
+        import numpy as np
+        return (np.random.randn(20, 6, 10, 10).astype(np.float32),)
```

Comment:
Could you also add tests for 3D and 5D inputs?

Reply:
added